scripts-graphql/rollup-tests.js

```diff
@@ -43,6 +45,10 @@ function envRollupInfo({browser, withDependencyTracking}) {
 
   if (browser) {
     plugins.unshift(globals(), builtins());
+    plugins.unshift(remap({
+      originalPath: './test-graphql/fetch-mock-node.js',
```

Comment:
Instead of always importing something that always has the name `node`, why not import a virtual file called `isomorphic-fetch-mock`, and when you're in browser mode, point that at the browser version, when you're in node mode, point it at the node version? This means there'd be complimentary stanzas in the if and else block here, but also that the naming in the import would be more consistent.